portal/src/hook/usePivot.ts

```diff
@@ -22,7 +22,8 @@ export function usePivotNavigation(
 
   useEffect(() => {
     if (!isHashValid(validItemKeys, hash)) {
-      navigate(`#${initialSelectedKey}`);
+      // NOTE: avoid adding extra entry to history stack
+      window.history.replaceState(undefined, "", `#${initialSelectedKey}`);
```

Comment:
This is incorrect. If we want to replace, `navigate` accepts options in the second argument. See the type definition for more details.

Reply:
@louischan-oursky fixed with `{replace: true}`